test/integration/datasets/DatasetsRepository.test.ts

```diff
@@ -35,57 +26,58 @@ describe('DatasetsRepository', () => {
     });
   });
 
-  describe('getDatasetById', () => {
-    test('should return dataset when it exists filtering by id', async () => {
-      const actual = await sut.getDatasetById(createdTestDatasetId);
-      expect(actual.id).toBe(createdTestDatasetId);
-    });
-
-    test('should return dataset when it exists filtering by id and version id', async () => {
-      const actual = await sut.getDatasetById(createdTestDatasetId, ':draft');
-      expect(actual.id).toBe(createdTestDatasetId);
-    });
+  describe('getDataset', () => {
+    describe('by numeric id', () => {
+      test('should return dataset when it exists filtering by id', async () => {
+        const actual = await sut.getDataset(TestConstants.TEST_CREATED_DATASET_ID);
+        expect(actual.id).toBe(TestConstants.TEST_CREATED_DATASET_ID);
```

Comment:
Because the response-to-model transformation is already tested in the unit test with all properties checked using mocked responses. I found no need to check the properties also here. In any case any error in the response-to-model transformation would raise an error both in the integration and unit test.